### DISCUSSION TITLE
How to upload google Tilt to Spoke?

### DISCUSSION THREAD

**Discussion by haonopolli** [mozilla/hubs#3344] [Spoke Questions] [answered]
Hi all, 

Are there any ways to locate contents I created using google Tilt in google poly and load them into spoke? I can't seem to be able to do that at the moment... Any advice would be really appreciated! 

Thank you! 

H

**Comment by pattersonbl2** [comment, marked as answer]
Google tilt should work in Hubs if the file was export or convert to an .glb before uploading .  We have found that the transparency tend to be problematic causing issues with how they look.
